Makefile

```diff
@@ -56,10 +56,11 @@ description = build/_output/description
 resources = deploy/namespace.yaml deploy/service_account.yaml deploy/role.yaml deploy/role_binding.yaml
 all: check handler
 
-check: format vet whitespace-check
+check: vet whitespace-check
+	test -z "`$(GOFMT) -l test/e2e/`"
```

Comment:
same as whitespace-check make a format-check Makefiel target, also add cmd/ and pkg/, also not that you have to add $(GO) as dependency so it install golang if it's not already there

In short:
```Makefile
gofmt-check: $(GO)
	test -z "`$(GOFMT) -l cmd/ pkg/ test/e2e/`"

check: vet whitespace-check gofmt-check
```